_quarto.yml

```diff
@@ -3,6 +3,7 @@ project:
   output-dir: docs
   render:
     - "*.qmd"
+    - "governance.md"
```

Comment:
one thing, why not just make it qmd?

Reply:
was a suggestion by @eitsupi - so people can view via GitHub with good formatting